CMakeLists.txt

```diff
@@ -113,9 +113,13 @@ install (TARGETS gencertchain RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 # This target requires ronn executable in your path.
 add_custom_target (
   man
-  COMMAND ronn -r ${CMAKE_SOURCE_DIR}/docs/gencertchain.md --pipe > ${CMAKE_SOURCE_DIR}/docs/gencertchain.1 &&
-          ronn -r ${CMAKE_SOURCE_DIR}/docs/gensxg.md --pipe > ${CMAKE_SOURCE_DIR}/docs/gensxg.1 &&
-          ronn -r ${CMAKE_SOURCE_DIR}/docs/libsxg.md --pipe > ${CMAKE_SOURCE_DIR}/docs/libsxg.3
+  COMMAND
+    ronn -r ${CMAKE_SOURCE_DIR}/docs/gencertchain.md --pipe >
+    ${CMAKE_SOURCE_DIR}/docs/gencertchain.1 && ronn -r
+    ${CMAKE_SOURCE_DIR}/docs/gensxg.md --pipe >
+    ${CMAKE_SOURCE_DIR}/docs/gensxg.1 && ronn -r
+    ${CMAKE_SOURCE_DIR}/docs/libsxg.md --pipe >
+    ${CMAKE_SOURCE_DIR}/docs/libsxg.3
```

Comment:
This wrapping makes the source harder to follow.  Consider using line continuations and indentation.

Reply:
I wished but this is the result of `cmake-format`.

Reply:
But why can't we make these formatting changes as part of a separate, smaller PR?